src/ucsi/ppm/mod.rs

```diff
@@ -119,6 +122,19 @@ impl Decode<CommandType> for Response {
     }
 }
 
+/// PPM command response
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub struct Response<T: PortId> {
+    /// CCI is produced by every command
+    pub cci: cci::Cci<T>,
+    /// Response data for the command
+    pub data: Option<ResponseData>,
+}
+
+pub type ResponseGlobal = Response<GlobalPortId>;
+pub type ResponseLocal = Response<LocalPortId>;
```

Comment:
Yoda naming
```suggestion
pub type GlobalResponse = Response<GlobalPortId>;
pub type LocalResponse = Response<LocalPortId>;
```

Elsewhere too.

Reply:
I think I got everything.